composer.json

```diff
@@ -1,41 +1,50 @@
 {
-	"name" : "goetas-webservices/xsd-reader",
-	"description" : "Read any XML Schema (XSD) programmatically with PHP",
-	"keywords" : [
-		"xsd",
-		"xml",
-		"php",
-		"parser",
-		"xmlschema"
-	],
-	"license" : "MIT",
-	"authors" : [{
-			"name" : "Asmir Mustafic",
-			"email" : "goetas@lignano.it"
-		}
-	],
-	"require" : {
-		"php" : ">=5.4.0"
-	},
-	"require-dev" : {
-		"phpunit/phpunit" : "^4.8|^5.0"
-	},
-	"autoload" : {
-		"psr-4" : {
-			"GoetasWebservices\\XML\\XSDReader\\" : "src/"
-		}
-	},
-	"autoload-dev" : {
-		"psr-4" : {
-			"GoetasWebservices\\XML\\XSDReader\\Tests\\" : "tests/"
-		}
-	},
-	"extra" : {
-		"branch-alias" : {
-			"dev-master" : "0.2-dev"
-		}
-	},
-	"scripts": {
-		"test": "vendor/bin/phpunit"
-	}
+    "name" : "goetas-webservices/xsd-reader",
+    "description" : "Read any XML Schema (XSD) programmatically with PHP",
+    "keywords" : [
+        "xsd",
+        "xml",
+        "php",
+        "parser",
+        "xmlschema"
+    ],
+    "license" : "MIT",
+    "authors" : [
+        {
+            "name" : "Asmir Mustafic",
+            "email" : "goetas@lignano.it"
+        }
+    ],
+    "config": {
+        "sort-packages": true
+    },
+    "require" : {
+        "php" : ">=5.6"
```

Comment:
since you changed this, why not use `^5.6||^7.0` ?

Reply:
afaik ^7.0 excludes hhvm, so it's a habit I've gotten into for people that choose to run it under hhvm.

If this pull request is successful, I'd be tempted to set it to `^5.6,<7.1` & use the php-7.1 branch exclusively for php 7.1.

Reply:
> afaik ^7.0 excludes hhvm, so it's a habit I've gotten into for people that choose to run it under hhvm

I'm not sure about it.. from what I know the HHVM version has an equivalent PHP version.
most of the open source packages uses `^5.6||^7.0`